packages/ckeditor5-dev-env/lib/release-tools/templates/template.hbs

```diff
@@ -1,5 +1,9 @@
 {{> header}}
 
+{{#if isInternalRelease}}
```

Comment:
I needed to duplicate the sentence because Handlebars does not support expression like this `{{#if condition1 || condition2}}`.